src/index.html

```diff
@@ -2,16 +2,171 @@
+
+        <label class="form-field" for="surname">Surname:
+          <input
+          type="text"
+          name="surname"
+          id="surname"
+          required
+          >
+        </label>
+
+        <label class="form-field" for="name">Name:
+          <input
+          type="text"
+          name="name"
+          id="name"
+          required
+          >
+        </label>
+
+        <label class="form-field" for="age">How old are You?
+          <input
+          type="number"
+          name="age"
+          id="age"
+          value="12"
+          min="1"
+          max="100"
+          >
+        </label>
+
+        <label class="form-field" for="birth">Full date of birth:
+          <input
+          type="date"
+          name="birth"
+          id="birth"
+          max="100"
+          >
+        </label>
+
+        <label class="form-field" for="agreement">I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="agreement"
+          id="agreement"
+          max="100"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="block">
+        <legend>Registration</legend>
+
+        <label class="form-field" for="email">E-mail:
+          <input
+          type="email"
+          name="email"
+          id="email"
+          placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="form-field" for="password">Password:
+          <input
+          type="password"
+          name="password"
+          id="password"
+          minlength="5"
+          maxlength="28"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="block">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">Do you love cats?
+          <input
+          type="radio"
+          name="radiobutton"
+          value="Yes"
+          id="yes"
+          >
+          <label for="yes">Yes</label>
+          <input
+          type="radio"
+          name="radiobutton"
+          value="No"
+          id="no"
+          >
+          <label for="no">No</label>
+        </div>
+
+        <label class="form-field" for="color">What is your favorite color?
+          <input type="color"
+          name="color"
+          id="color"
+          >
+        </label>
+
+        <label class="form-field" for="time">What time do you go to bed?
+          <input
+          type="time"
+          name="time"
+          id="time"
+          >
+        </label>
+
+        <label class="form-field" for="select">What are your favorite brand of cars?
+          <select
+          name="select"
+          id="select"
+          multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field" for="input">How do you rate our work?
```

Comment:
It'd be better to move text to a new line. Make sure to fix it everywhere
```suggestion
        <label class="form-field" for="input">
        	How do you rate our work?
```